sql/item_mods.sql

```diff
@@ -8131,7 +8131,7 @@ INSERT INTO `item_mods` VALUES (11671,25,3); -- ACC: 3
 
 -- Mujin Band
 INSERT INTO `item_mods` VALUES (11672,175,5); -- SKILLCHAINDMG: 5
-INSERT INTO `item_mods` VALUES (11672,487,5); -- MAG_BURST_BONUS: 5
+-- TODO: MAG_BURST_BONUS_II: 5
```

Comment:
Could you expand on this?

I ask because I could add support for it with enough info.

Reply:
I think the only difference between the Magic Burst Damage and Magic Burst Damage II is:
Exceeds the 40% [Magic Burst](https://www.bg-wiki.com/ffxi/Magic_Burst) cap